core/src/main/java/org/elasticsearch/cluster/routing/AllocationId.java

```diff
@@ -148,12 +156,52 @@ public String toString() {
 
     @Override
     public XContentBuilder toXContent(XContentBuilder builder, Params params) throws IOException {
-        builder.startObject("allocation_id");
-        builder.field("id", id);
+        builder.startObject(ALLOCATION_ID_OBJECT_KEY);
```

Comment:
This what we do in all other classes. If you don't like it, fine, but can you say why so we'll stop doing it in other places?